samples/getting_started/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import {v2 as cloudinary} from 'cloudinary';
+
+// Update to your credentials
+cloudinary.config({
+    cloud_name: '<cloud_name>',
+    api_key: '<api_key>',
+    api_secret: '<api_secret>'
+});
+
+// Ensure that you're registered to auto tagging https://cloudinary.com/documentation/google_auto_tagging_addon
+// Ensure that you're registered to background removal https://cloudinary.com/documentation/cloudinary_ai_background_removal_addon
+//Upload an image with auto tag
+//apply eager transformation
+cloudinary.uploader.upload(
+    "https://res.cloudinary.com/demo/image/upload/v1707306308/cld-docs-hp/walking_woman",
+    {
+        public_id: "walking_woman",
+        categorization: "google_tagging",
```

Comment:
Just need to add the auto-tagging param in addition to categorization to instruct the add-on to actually add tags.  The categorization just tells the add-on to return all the categories it finds.  The `auto-tagging: 0.8` then tells the add-on to find all the 'categories'  that were returned with a confidence level 0.8 or higher and add them as tags.

(I randomly picked 0.8 - we might want to pick 0.9 or 0.95 even for our needs to ensure we get the `overcoat` value, and only the next 2 or 3 additional tags at most - and then the designer should use exactly those values in the image we show.